### ISSUE TITLE
[BUG] Selected Submission Sheet: Tab key navigation doesn't work as expected

### ISSUE THREAD

**Issue by PeterMarinov** [endatix/endatix#225]
**Describe the bug**
The tab key on the Selected Submission Sheet is working in a very strange way and is not usable for visually impaired people or just keyboard navigation

**To Reproduce**

1. Go to Submission List page
2. Click on a submission row to open the Selected Submission Sheet
3. Start pressing the Tab key on the keyboard
4. Observe the focus zone behavior and note the following issues:
   - It takes 2 key presses to move to the next element

**Expected behavior**
Tab navigation starts from the first element in the DOM and continues in linear fashion. Also it should take one Tab press to move to the next button.

**Screenshots**
![image](https://github.com/user-attachments/assets/a82624d7-0b1b-4be0-ae23-1f02a36fb14d)

**Desktop (please complete the following information):**
 - N/A
